example/bundle-cjs.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
You should add to the `example/package.json` these scripts:

```json
  "scripts": {
    "start": "npm run start:cjs && npm run start:esm",
    "start:cjs": "node bundle-cjs.js",
    "start:esm": "node bundle-esm.mjs"
  },
```

current scripts doesn't work.

Reply:
Added these, plus
```js
"start:ts": "ts-node bundle.ts"
```